src/dmd/dmodule.d

```diff
@@ -1565,6 +1564,16 @@ extern (C++) final class Module : Package
             buf.prependstring(package_.ident.toChars());
         }
     }
+
+    /** Lazily initializes and returns the escape table.
+    Turns out it eats a lot of memory.
+    */
+    Escape* escapetable()
```

Comment:
Just mark this as `extern (D)` and we're all good.

Reply:
done, thanks